tests/integration_test_net.go

```diff
@@ -289,6 +350,13 @@ func startIntegrationTestNet(
 	// the network's session needs to know about the network itself
 	net.net = net
 
+	heapProfile := os.Getenv(heapProfileEnvVar)
+	if heapProfile == "1" ||
+		strings.EqualFold(heapProfile, "on") ||
+		strings.EqualFold(heapProfile, "true") {
+		go startHeapProfiler(t)
+	}
```

Comment:
nit: may be moved into the function since any re-use will have to do the same.

Reply:
done